newIDE/app/src/ProjectManager/PublicGameProperties.js

```diff
@@ -56,6 +69,29 @@ function PublicGameProperties({
           </Trans>
         }
       />
+      {// This view is used for public game properties as well as project properties.
+      // The following properties are not shown in project properties.
+      setPlayableWithKeyboard &&
+        setPlayableWithGamepad &&
+        setPlayableWithMobile && (
+          <React.Fragment>
+            <Checkbox
+              label={<Trans>Playable with keyboards</Trans>}
+              checked={!!playWithKeyboard}
+              onCheck={(e, checked) => setPlayableWithKeyboard(checked)}
+            />
+            <Checkbox
+              label={<Trans>Playable with gamepads</Trans>}
```

Comment:
same here, without s? (yes multiple gamepads can be used, but keeping the generic word "gamepad" should mean that multiple can be used)